### DISCUSSION TITLE
Error after install ver 2.5.144 [Refused to apply style because its MIME type ('text/plain') is not a supported stylesheet MIME type]

### DISCUSSION THREAD

**Discussion by EbiPenMan** [requarks/wiki#3445] [Help / Questions]
OS: CentOS Linux 7 -  3.10.0-862.11.6.el7.x86_64 , Docker
Wiki.js version:  2.5.144
Database engine: postgres:11-alpine

I run wiki.js with docker-compose, with image **ver [image: requarks/wiki:2] it's OK.** But with ver [image: requarks/wiki:2.5.144]
I have a problem after installation.

`Refused to apply style from 'http://---:3000/_assets/css/app.3faf3def5b1f890b8592.css' because its MIME type ('text/plain') is not a supported stylesheet MIME type, and strict MIME checking is enabled.`

![0](https://user-images.githubusercontent.com/7556608/93751507-10cd8b80-fc0a-11ea-9ee7-914f13206df7.png)
![1](https://user-images.githubusercontent.com/7556608/93751531-1925c680-fc0a-11ea-810b-d4ef98ab6619.png)
![2](https://user-images.githubusercontent.com/7556608/93751544-1d51e400-fc0a-11ea-9d04-919128a61abe.png)
![3](https://user-images.githubusercontent.com/7556608/93751551-1f1ba780-fc0a-11ea-98b1-99c51a81cf22.png)
![4](https://user-images.githubusercontent.com/7556608/93751559-217e0180-fc0a-11ea-8b5e-8c4ea3494f15.png)
![5](https://user-images.githubusercontent.com/7556608/93751568-2478f200-fc0a-11ea-86a1-fd7abd86df95.png)

**Comment by EbiPenMan** [comment]
In these 3 days, there was no one to give me an answer?

**Comment by Smankusors** [comment]
hmm can you open the URLs mentioned on the DevTools?

**Comment by EbiPenMan** [comment]
> hmm can you open the URLs mentioned on the DevTools?

thank you for your replay, yes I can open all links in the new tab and no problem when direct open these files.

**Comment by EbiPenMan** [comment]
I'm waiting for a solution.

**Comment by Smankusors** [comment]
if you can open those files fine, then there's no reason it's failing to load with "error not found".

Other than that, I have no idea...
